sdk/core/core-compat/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "@azure/core-compat",
+  "version": "1.0.0",
+  "description": "Core Shim Library to bridge the gap between Core V1 & V2 packages.",
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/index.js",
+  "types": "types/latest/core-compat.d.ts",
+  "typesVersions": {
+    "<1.0": {
+      "types/latest/*": [
+        "types/1.0/*"
+      ]
+    }
```

Comment:
Doesn't this mean "typescript versions less than 1.0"? Seems like this must be an error.

Reply:
Removed it